src/config.js

```diff
@@ -135,6 +135,17 @@ module.exports.configs = {
         'prefer-template': 'error',
         'no-param-reassign': 'off', // We don't use `arguments`, and assigning to parameters can be useful
         'no-promise-executor-return': 'off', // Allow implicit return in promise executor
+        'capitalized-comments': [
+          'error',
+          'always',
+          {
+            ignorePattern:
+              'pragma|ignore|prettier-ignore|webpack\\w+:|c8|return|const|let|var|await|function|console',
```

Comment:
I added the part starting with `return`. The rest of it was already there, from eslint-config-xo: https://github.com/xojs/eslint-config-xo/blob/main/index.js#L281